src/types/any.rs

```diff
@@ -1127,6 +1127,15 @@ pub trait PyAnyMethods<'py>: crate::sealed::Sealed {
     where
         O: ToPyObject;
 
+    /// Computes the negative of self or `-self`.
```

Comment:
I slightly prefer a longer form of the wording here, which I think can also be used for the others.

```suggestion
    /// Computes the negative of self.
    ///
    /// Equivalent to the Python expression `-self`.
```